### PR TITLE
Warning to connect to remote server with Instance name

### PR DESCRIPTION
**Pull request by saitorhan**

### CODE REVIEW

**Comment by Court72** [comment]
@MashaMSFT 

Can you review the proposed changes? *IMPORTANT*: When the changes are ready for publication, add a `#sign-off` comment to signal that the PR is ready for the review team to merge. 

#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

**Comment by saitorhan** [comment]
Dear @Court72, I have been using SQL Server for a long time and I will try to add my experiences and fine points that I use in the titles one by one.
Thank you for your support 🙂

File: docs/relational-databases/lesson-2-connecting-from-another-computer.md [hunk 1]
```diff
@@ -121,6 +121,9 @@ When the [!INCLUDE[ssNoVersion](../includes/ssnoversion-md.md)] Browser service
   
     > [!NOTE]  
     > If you omit **tcp:** from the **Server name** box, then the client will attempt all protocols that are enabled, in the order specified in the client configuration.  
+
+    > [!IMPORTANT]
+    > If an attempt is made to establish a connection with the Instance name while connecting to the remote server, the [SQL Server Browser](../tools/configuration-manager/sql-server-browser-service.md) service must be running. Instance name-port mapping will not work if the SQL Server Browser service is not running.
```
Comment (MashaMSFT) [inline docs/relational-databases/lesson-2-connecting-from-another-computer.md:126]:
```suggestion
    > If an attempt is made to establish a connection with the instance name while connecting to the remote server, the [SQL Server Browser](../tools/configuration-manager/sql-server-browser-service.md) service must be running on the remote server. Instance name port mapping does not work if the SQL Server Browser service is not running.

```

File: docs/relational-databases/lesson-2-connecting-from-another-computer.md [hunk 2]
```diff
@@ -121,6 +121,10 @@ When the [!INCLUDE[ssNoVersion](../includes/ssnoversion-md.md)] Browser service
   
     > [!NOTE]
```
Comment (MashaMSFT) [inline docs/relational-databases/lesson-2-connecting-from-another-computer.md:122]:
```suggestion
    > [!IMPORTANT]  

```

File: docs/relational-databases/lesson-2-connecting-from-another-computer.md [hunk 3]
```diff
@@ -121,6 +121,10 @@ When the [!INCLUDE[ssNoVersion](../includes/ssnoversion-md.md)] Browser service
   
     > [!NOTE]  
     > If you omit **tcp:** from the **Server name** box, then the client will attempt all protocols that are enabled, in the order specified in the client configuration.
```
Comment (MashaMSFT) [inline docs/relational-databases/lesson-2-connecting-from-another-computer.md:123]:
```suggestion
    > - If you omit **tcp:** from the **Server name** box, then the client will attempt all protocols that are enabled, in the order specified in the client configuration.  
    > - If an attempt is made to establish a connection with the instance name while connecting to the remote server, the [SQL Server Browser](../tools/configuration-manager/sql-server-browser-service.md) service must be running on the remote server. Instance name port mapping does not work if the SQL Server Browser service is not running.

```

File: docs/relational-databases/lesson-2-connecting-from-another-computer.md [hunk 4]
```diff
@@ -121,6 +121,10 @@ When the [!INCLUDE[ssNoVersion](../includes/ssnoversion-md.md)] Browser service
   
     > [!NOTE]  
     > If you omit **tcp:** from the **Server name** box, then the client will attempt all protocols that are enabled, in the order specified in the client configuration.  
+
+    > [!IMPORTANT]
```
Comment (MashaMSFT) [inline docs/relational-databases/lesson-2-connecting-from-another-computer.md:125]:
```suggestion
```

File: docs/relational-databases/lesson-2-connecting-from-another-computer.md [hunk 5]
```diff
@@ -121,6 +121,10 @@ When the [!INCLUDE[ssNoVersion](../includes/ssnoversion-md.md)] Browser service
   
     > [!NOTE]  
     > If you omit **tcp:** from the **Server name** box, then the client will attempt all protocols that are enabled, in the order specified in the client configuration.  
+
+    > [!IMPORTANT]
+    > If an attempt is made to establish a connection with the instance name while connecting to the remote server, the [SQL Server Browser](../tools/configuration-manager/sql-server-browser-service.md) service must be running on the remote server. Instance name port mapping does not work if the SQL Server Browser service is not running.
```
Comment (MashaMSFT) [inline docs/relational-databases/lesson-2-connecting-from-another-computer.md:126]:
```suggestion
```

**Review by MashaMSFT — COMMENTED** [review]
combining notes

File: docs/relational-databases/lesson-2-connecting-from-another-computer.md [hunk 6]
```diff
@@ -119,8 +119,13 @@ When the [!INCLUDE[ssNoVersion](../includes/ssnoversion-md.md)] Browser service
   
 3.  In the **Server name** box, type **tcp:** to specify the protocol, followed by the computer name, a comma, and the port number. To connect to the default instance, the port 1433 is implied and can be omitted; therefore, type **tcp:**_<computer_name>_. In our example for a named instance, type **tcp:**_<computer_name>_**,49172**.  
   
-    > [!NOTE]  
-    > If you omit **tcp:** from the **Server name** box, then the client will attempt all protocols that are enabled, in the order specified in the client configuration.  
+    > [!IMPORTANT]  
+
```
Comment (MashaMSFT) [inline docs/relational-databases/lesson-2-connecting-from-another-computer.md:123]:
```suggestion
```

**Comment by MashaMSFT** [comment]
Hi @saitorhan , thanks so much for your submission! I modified the text slightly, and combined the two notes :) Thanks again! #sign-off

**Pull request merged**